spikeinterface/preprocessing/whiten.py

```diff
@@ -23,8 +26,15 @@ class WhitenRecording(BasePreprocessor):
         Random seed for random chunk, by default None
     W : 2d np.array
         Pre-computed whitening matrix, by default None
-    M : 1d np.array
-        Pre-computed means
+    M : 1d np.array or None
+        Pre-computed means.
+        If M is None then it is equivalent to apply_mean=False
+    apply_mean: bool
+        Substract or not the mean matrix M before the dot product with W.
+    int_scale : None or float
+        Apply a scale after the whetening.
+        This is usefull for dtype=int and want to keep the output as int also typical scale are 200 which means
+        that the noise in noise [0-1] will be encoded in [0-200].
```

Comment:
same as above

Reply:
@samuelgarcia can you propagate the docs changes here?

Reply:
done